scalafmt-tests/src/test/resources/rewrite/SortImports.stat

```diff
@@ -35,8 +44,8 @@ import a.{
   bar
 }
 >>>
-import a.{zzzz => _, bar}
-<<< caveat: comments
+import a.{bar, zzzz => _}
```

Comment:
@olafurpg I "fixed" this. I say "fixed" because there was a comment about "being sure this works". I don't see why it wouldn't so I just changed it.